README.md

```diff
@@ -32,6 +32,18 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+## Adding Trello credentials
+1. Create an account on https://trello.com/signup.
+2. Generate an API key and token by following the instructions https://trello.com/app-key.
+4. Create ```config.py``` file in project root.
+3. Copy the key a token and paste in config.py file in this format: 
+
+```
+class APIConfig():
+    KEY = "your key"
+    TOKEN = "your token"
+```
```

Comment:
It's nicer to use .env rather than a gitignored python file. You can keep going with the config.py pattern but you will find it more difficult in future modules. In fact you will probably find you want to make the switch in module 3 because the instructions assume you're using .env.